### DISCUSSION TITLE
Questions about where to find the data of variables such as grains.subBoundaryLength, grains.innerBoundary.misorientation.angle?

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#923] [Ask Anything]
Hi, 
I am a beginner working on mtex and I wish to know this great toolbox very well.
In the online documentations I find that there are some variables like _grains.innerBoundary.misorientation.**angle**_, _grains.**subBoundaryLength**_,_grains.boundary.**curvature**_. When I input them into command window, corresponding data shows. But why I can not find them in the workspace?

**Comment by Brian-Liu-rx** [comment]
![image](https://user-images.githubusercontent.com/69452222/113101515-6628cc00-922f-11eb-9498-19a0f9a97381.png)

An example: There is no _grains.innerBoundary.misorientation.angle_in the matlab workspace

**Reply by kilir** [reply]
Hi,
the reason is, that those are either properties or functions of the object itself. If you want to write it to a separate variable, `abc = grains.something` .
Cheers,
Rüdiger

**Reply by Brian-Liu-rx** [reply]
Hi, Rüdiger,
    I understand now. 
    Another one is I am curious about how the curvature  in grain boundary is calculated by mtex. Is there any paper or documentation about the algorithm of it?
    Thanks!
    Biran
